torchreid/integration/sc/inference_task.py

```diff
@@ -272,16 +276,16 @@ def export(self, export_type: ExportType, output_model: ModelEntity):
                 export_onnx(self._model.eval(), self._cfg, onnx_model_path)
                 self._model.mix_precision = mix_precision_status
                 export_ir(onnx_model_path, self._cfg.data.norm_mean, self._cfg.data.norm_std,
-                          optimized_model_dir=optimized_model_dir, data_type=optimized_model_precision.name)
+                          optimized_model_dir=optimized_model_dir)
```

Comment:
Is it ok that 'data_type=optimized_model_precision.name' removed?

Reply:
Yes, the default value is FP32.